.github/workflows/build.yml

```diff
@@ -39,3 +36,73 @@ jobs:
             ~/repo-cache
             ~/disk-cache
           key: repo-cache-${{ runner.os }}-nixpkgs-${{ env.cache-version }}-${{ github.run_id }}-${{ github.run_attempt }}
+
+  run-tests-with-darwin:
+    name: Run Tests with darwin runner
+    runs-on: macos-11
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Install NixOS
+      uses: cachix/install-nix-action@v18
+      with:
+        nix_path: nixpkgs=./nixpkgs.nix
+
+    - name: Install cachix
+      uses: cachix/cachix-action@v10
+      with:
+          name: tweag
+
+    - name: Run cachix
+      shell: bash
+      run: |
+        set -e -o pipefail
+        cachix use tweag
+        cachix push tweag watch-store &
+
+    - name: Configure
+      run: |
+        mkdir -p ~/repo-cache ~/disk-cache
```

Comment:
@facundominguez 
is this the right configuration for darwin/macos
` echo build --host_platform=@io_tweag_rules_nixpkgs//nixpkgs/platforms:host > .bazelrc.local`